CHANGES.md

```diff
@@ -1,8 +1,76 @@
 # Change Log / Release Notes
 
-## 0.13
+## 0.13 (Sometime Early 2017)
 
-  * Re-builds using script `<xmonaddir>/build` if it exists and is executable
+### Breaking Changes
+
+  * When restarting xmonad, resume state is no longer passed to the
+    next process via the command line.  Instead, a temporary state
+    file is created and xmonad's state is serialized to that file.
+
+    When upgrading to 0.13 from a previous version, the `--resume`
+    command line option will automatically migrate to a state file.
+
+    This fixes issue #12.
+
+### Enhancements
+
+  * You can now control which directory xmonad uses for finding your
+    configuration file and which one is used for storing the compiled
+    version of your configuration.  In order of preference:
+
+      1. New environment variables.  If you want to use these ensure
+         you set the correct environment variable and also create the
+         directory it references:
+
+         - `XMONAD_CONFIG_DIR`
+         - `XMONAD_CACHE_DIR`
+         - `XMONAD_DATA_DIR`
+
+      2. XDG Base Directory Specification directories, if they exist:
+
+         - `XDG_CONFIG_HOME/xmonad`
+         - `XDG_CACHE_HOME/xmonad`
+         - `XDG_DATA_HOME/xmonad`
+
+      3. The `~/.xmonad` directory.
+
+    If none of these directories exist then (1) will be used if it is
+    set, otherwise (2) will be used.  Of course, if neither (1) nor
+    (2) exist but (3) does then (3) will be used just like previous
+    versions of xmonad.
+
+    This fixes a few issues, notably #7 and #56.
+
+  * A custom build script can be used when xmonad is given the
+    `--recompile` command line option.  If an executable named `build`
+    exists in the xmonad configuration directory it will be called
+    instead of `ghc`.  It takes one argument, the name of the
+    executable binary it must produce.
+
+    Note: the build script is called whenever `ghc` would have been
+    called.  That means the `xmonad.hs` file (or any files in `lib`
+    need to have newer time stamps than the generated executable in
+    order for the build script to be called.
+
+    This is one possible fix for issue #8.
```

Comment:
This kind of makes it sound like we haven't decided which of two fixes we are going to use for #8.  But if I understand correctly, what you really mean is that there are now two mechanisms by which a user can solve the problem described in #8.  Is that correct?  If so, maybe a better way to phrase it would be something like "This is one of two mechanisms that can be used to achieve sandboxed xmonad builds (see #8)"?